gradle/dokka-multiproject-gradle-example/projectB/build.gradle

```diff
@@ -0,0 +1,10 @@
+dependencies {
+    compile project(":projectA")
+}
+
+dokka {
+    externalDocumentationLink {
+        url = new URL("file://${project(":projectA").buildDir}/dokka/project-a/")
+        packageListUrl = new URL(url, "package-list")
```

Comment:
Does this work when publishing this documentation online? Does this create relative URLs? 🤔

Reply:
Well actually that depends. Some links are created as relative, regardless of this config, but some are not. For example the "all types" page is created with absolute paths, but the links in specific class pages are created with relative paths. I think it'll change in future versions of Dokka. 
As for the publishing - I think you can set the URL to your domain (http://example.com/...) and create package list URL with local path (file://projectname/...) and that should work. 
I didn't test it, though